.gitlab/ci/bucket-upload.yml

```diff
@@ -168,7 +174,7 @@ upload-packs-to-marketplace:
         STORAGE_BASE_PATH="content"
       fi
       if [[ $GCS_MARKET_BUCKET != $GCS_PRODUCTION_BUCKET ]] && [[ -z $STORAGE_BASE_PATH ]]; then
-        STORAGE_BASE_PATH="upload-flow/builds/$CI_COMMIT_BRANCH/$CI_PIPELINE_ID/xsoar/content"
+        STORAGE_BASE_PATH="upload-flow/builds/$CI_COMMIT_BRANCH/$CI_PIPELINE_ID/content"
       fi
 
       PACKS_SRC="gs://$GCS_MARKET_BUCKET/$STORAGE_BASE_PATH/packs"
```

Comment:
We don't need to override the variables in the `force-pack-upload`?

Reply:
we probably should, as far as I remember, there are a few problems with that flow at the moment, and I didn't want to go down the rabbit hole